CHANGELOG.md

```diff
@@ -4,3 +4,4 @@
 
 - Add wgpu triangle example ([#53] by [@lord])
 - Add thread-safe app handle ([#90] by [@clavin])
+- Updated AccessKit to 0.11 ([#108] by [@waywardmonkeys])
```

Comment:
Unfortunately, these links do not get added automatically; could you add them below? (The previous two  changelog entries are broken, so don't follow their example)

Reply:
Done. Once this lands, I don't mind submitting a second PR to fix the links for the others.